content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -19,6 +19,7 @@ For example, using the .NET CLI:
 
 ```shell
 dotnet add package OpenTelemetry
+dotnet add package OpenTelemetry.Exporter.Console
```

Comment:
this whole section feels unnecessary....

Reply:
you are right.